index.html

```diff
@@ -10,14 +10,10 @@
       <div class="col-md-6 space-below">
         <p>
           <b>English: </b> <a href="http://www.clojurebridge.org/">ClojureBridge</a>
-          is a free Clojure programming workshop for women*.
-          Especially for those with just a little or no programming experience.
-          <b>The next workshop is on Friday June 17, and Saturday June 18, 2016.</b>
-        </p>
-        <p>
-          * We are offing this workshop to help make the Clojure community more accessible 
-          to a diverse range of participants. As such, registration is open to people who 
-          identify as a woman or have a nonbinary gender identity.
+          is a free Clojure programming workshop for women*. Especially for those with
+          just a little or no programming experience.
```

Comment:
I know this is not a new change, but actually this might be better as:
> is a free Clojure programming workshop for women* **— **especially for those **with little to no** programming experience